hw_diag/app.py

```diff
@@ -46,6 +47,11 @@ def get_app(name):
     app = Flask(name)
 
     cache.init_app(app)
+    
+    try:
+        ensure_quectel_health()
+    except Exception as e:
+        log.error('Error in modem settings: {}'.format(e))
```

Comment:
Error message seems misleading. We aren't sure exactly what causes an exception caught here.

Reply:
Yes this is catch all. What do you suggest?

Reply:
`Unknown error encountered while trying to update Quectel modems for network compatibility`